src/test/java/seedu/internhub/logic/parser/InternHubParserTest.java

```diff
@@ -1,41 +1,41 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertTrue;
-import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
-import static seedu.address.logic.Messages.MESSAGE_UNKNOWN_COMMAND;
-import static seedu.address.testutil.Assert.assertThrows;
-import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
+import static seedu.internhub.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.internhub.logic.Messages.MESSAGE_UNKNOWN_COMMAND;
+import static seedu.internhub.testutil.Assert.assertThrows;
+import static seedu.internhub.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
 
 import java.util.Arrays;
 import java.util.List;
 
 import org.junit.jupiter.api.Test;
 
-import seedu.address.commons.core.index.Index;
-import seedu.address.logic.commands.AddCommand;
-import seedu.address.logic.commands.ClearCommand;
-import seedu.address.logic.commands.DeleteCommand;
-import seedu.address.logic.commands.EditCommand;
-import seedu.address.logic.commands.ExitCommand;
-import seedu.address.logic.commands.FilterCommand;
-import seedu.address.logic.commands.FindCommand;
-import seedu.address.logic.commands.HelpCommand;
-import seedu.address.logic.commands.ListCommand;
-import seedu.address.logic.commands.NoteCommand;
-import seedu.address.logic.commands.ViewCommand;
-import seedu.address.logic.parser.exceptions.ParseException;
-import seedu.address.model.person.MatchingTagPredicate;
-import seedu.address.model.person.NameContainsKeywordsPredicate;
-import seedu.address.model.person.Person;
-import seedu.address.testutil.EditPersonDescriptorBuilder;
-import seedu.address.testutil.PersonBuilder;
-import seedu.address.testutil.PersonUtil;
-
-
-public class AddressBookParserTest {
-
-    private final AddressBookParser parser = new AddressBookParser();
+import seedu.internhub.commons.core.index.Index;
+import seedu.internhub.logic.commands.AddCommand;
+import seedu.internhub.logic.commands.ClearCommand;
+import seedu.internhub.logic.commands.DeleteCommand;
+import seedu.internhub.logic.commands.EditCommand;
+import seedu.internhub.logic.commands.ExitCommand;
+import seedu.internhub.logic.commands.FilterCommand;
+import seedu.internhub.logic.commands.FindCommand;
+import seedu.internhub.logic.commands.HelpCommand;
+import seedu.internhub.logic.commands.ListCommand;
+import seedu.internhub.logic.commands.NoteCommand;
+import seedu.internhub.logic.commands.ViewCommand;
+import seedu.internhub.logic.parser.exceptions.ParseException;
+import seedu.internhub.model.person.MatchingTagPredicate;
+import seedu.internhub.model.person.NameContainsKeywordsPredicate;
+import seedu.internhub.model.person.Person;
+import seedu.internhub.testutil.EditPersonDescriptorBuilder;
+import seedu.internhub.testutil.PersonBuilder;
+import seedu.internhub.testutil.PersonUtil;
+
+
+public class InternHubParserTest {
+
+    private final InternHubParser parser = new InternHubParser();
```

Comment:
Thank you for updating all references from AddressBook to InternHub. This adjustment aligns perfectly with our product's identity as InternHub, ensuring consistency and clarity throughout the application.